internal/pkg/build/build.go

```diff
@@ -59,13 +59,13 @@ type Config struct {
 }
 
 // NewBuild creates a new Build struct from a spec (URI, definition file, etc...)
-func NewBuild(spec string, conf Config) (*Build, error) {
+func NewBuild(spec string, conf Config, allowUnauthenticatedBuild, localVerifyBuild bool) (*Build, error) {
```

Comment:
Why not pass these as parameters of `Config`?

Reply:
Good idea, I'll fix that up. *Back then when I made this PR my go knowledge was less then is is now.*

Reply:
And yes, I know the warning messages need to be fixed. If you have suggestions that will help :+1: